examples/bazel_managed_deps/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
npm_install not used here as it would over-complicate the example to have both npm_install and yarn_install. I added `internal/e2e/fine_grained_deps` to test the npm_install case